backend/views/base.pug

```diff
@@ -5,9 +5,10 @@ html
     link(rel='stylesheet', href='https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css')
     link(rel='stylesheet', href='https://fonts.googleapis.com/css?family=Open+Sans&display=swap')
     link(rel='stylesheet', href='/stylesheets/style.css')
-    
+
+    script(src='https://code.jquery.com/jquery-3.4.1.min.js')
```

Comment:
Are we adding jquery for `ajax` requests only here?

Reply:
What do you mean by `only here`?

Yes, the main purpose of using `jquery` is to use its `ajax` functionality.
but we can use it's other features once the development goes further.